### PR TITLE
Mainnet mellow deployments

### PR DESCRIPTION
**Pull request by 0xZenus**
Testnet URL: https://feat-mainnet-mellow-deployments.d2f2fhja7tsifl.amplifyapp.com/#/trader-pools

### CODE REVIEW

File: src/routes/Ecosystem/Common/VaultField.tsx [hunk 1]
```diff
@@ -3,32 +3,71 @@ import { Typography } from '@components/atomic';
 import { Box } from '@mui/system';
 import React from 'react';
 import { titleStyles } from './styles';
-import { ReactComponent as Mellow } from '../mellow-icon.svg';
+import { ReactComponent as USDC } from './usdc-icon.svg';
+import { ReactComponent as ETH } from './eth-icon.svg';
+import { ReactComponent as DAI } from './dai-icon.svg';
+import { ReactComponent as USDT } from './usdt-icon.svg';
+
+const getTokenIcon = (token: string) => {
```
Comment (ssbarbee) [inline src/routes/Ecosystem/Common/VaultField.tsx:11]: dont we already have these in the application? All you need to do is <Icon name="token-dai" /> iirc? Search for such usage. I don't think we need new set of icons but I might be wrong

Comment (0xZenus) [reply src/routes/Ecosystem/Common/VaultField.tsx:11]: They have different backgrounds.

Comment (ssbarbee) [reply src/routes/Ecosystem/Common/VaultField.tsx:11]: I see thanks!

File: src/components/interface/Nav/Nav.tsx [hunk 2]
```diff
@@ -42,11 +42,16 @@ const Nav: React.FunctionComponent = React.memo(() => (
...
 
     <NavLink
+      isNew={true}
       subLinks={[
         {
           text: 'LP POOLS',
           link: `/${routes.POOLS}`,
         },
+        {
+          text: 'LP OPTIMIZERS',
```
Comment (Sol3iman) [inline src/components/interface/Nav/Nav.tsx:52]: Minor detail but the copy everywhere else spells Optimiser with an s. Good to be consistent with this.

File: src/hooks/useMellowLPVaults/config.ts [hunk 3]
```diff
@@ -24,154 +22,208 @@ const networkConfigurations: { [key: string]: NetworkConfiguration } = {
...
+          estimatedHistoricApy: 's00n',
+          description: 'Optimised for LPing across pools in 50x levered positions.',
+          underlyingPools: ['AAVE - ETH BORROW'],
+          show: true,
+          soon: true,
+          deprecated: false,
+        },
+      },
+      {
+        router: '0x0000000000000000000000000000000000000000',
```
Comment (Sol3iman) [inline src/hooks/useMellowLPVaults/config.ts:123]: Is this because Mellow do not support USDT?

Comment (0xZenus) [reply src/hooks/useMellowLPVaults/config.ts:123]: yes.

Comment (Sol3iman) [reply src/hooks/useMellowLPVaults/config.ts:123]: Thanks. Could you share the testing url in the description as well, please?

File: src/components/interface/Nav/Nav.tsx (hunk 2 above)
Comment (Sol3iman) [reply src/components/interface/Nav/Nav.tsx:52]: Thank you

**Review by ssbarbee — APPROVED** [review]
LGTM

**Review by arturbeg — APPROVED** [review]

File: src/hooks/useMellowLPVaults/config.ts (hunk 3 above)
Comment (0xZenus) [reply src/hooks/useMellowLPVaults/config.ts:124]: Done

**Pull request merged**